source/node/Westley.js

```diff
@@ -64,6 +64,10 @@ class Westley extends EventEmitter {
         return this._dataset;
     }
 
+    get entryChanges() {
+        // return [...this._entryChanges];
```

Comment:
extra comment here

Reply:
Thanks! This should be removed. Will check now.